.claude/skills/pr-review-resolver/scripts/post_comments.py

```
@@ -0,0 +1,127 @@
+        print(f"Error: Failed to parse {path}: {e}")
+        return None
+    if not isinstance(data, dict):
+        print(f"Error: {path} must be a mapping")
+        return None
+    return data
+
+
+def get_pr_number() -> int | None:
+    """현재 브랜치의 PR 번호를 가져온다."""
+    result = subprocess.run(
+        ["gh", "pr", "view", "--json", "number", "-q", ".number"],
```

Ruff (0.14.11)

Starting a process with a partial executable path

(S607)